src/baldr/graphtile.cc

```diff
@@ -126,24 +129,51 @@ GraphTile::GraphTile(const GraphId& graphid, char* ptr, size_t size) : header_(n
   Initialize(graphid, ptr, size);
 }
 
-GraphTile::GraphTile(const std::string& tile_url, const GraphId& graphid, curler_t& curler) {
+GraphTile GraphTile::CacheTileURL(const std::string& tile_url,
```

Comment:
made this into a static function that returns a graphtile instead of a constructor. its actually probably not necessary to be honest but i thought because of the caching to disk function it was a bit less tricky or more obvious to give it a name. let me know if anyone thinks this is better off as a constructor.